.github/workflows/zip_studies.yml

```diff
@@ -0,0 +1,61 @@
+name: Zip studies on push to master branch
+on:
+  # Trigger the workflow on push,
+  # but only for the master branch
+  push:
+    branches:
+      - master
+    paths:
+      - 'public/**'
+jobs:
+  zip_studies:
+    if: github.repository == 'cbioportal/datahub'
+    runs-on: ubuntu-latest
+    timeout-minutes: 1440
+    steps:
+    - id: files
+      uses: jitterbit/get-changed-files@v1
+    - run: | # TODO remove step
+        echo ${{ steps.files.outputs.all }}
+        exit 0
+    - run: |
+        echo "##[set-output name=unique_studies_list;]$(echo "${{ steps.files.outputs.all }}" | awk 'BEGIN { RS=" "; FS="/"; ORS=" " } { set[$2]++ } END { for (dir in set) { print dir } }')"
+        exit 0
+      id: unique_studies
+    - run: | # TODO remove step
+        echo ${{ steps.unique_studies.outputs.unique_studies_list }}
+        exit 0
+    - run: |
+        git --version
+        exit 0
+    - uses: actions/checkout@v2
+    #  with:
```

Comment:
maybe remove these two commented lines? Or, put them in with "lfs: 'false'" if that is appropriate.